libs/frontend/feature-site-translations/src/lib/use-site-translations.ts

```diff
@@ -129,7 +129,12 @@ export const useSiteTranslations = (): ISiteTranslationsFeature => {
     if (edit.key !== edit.originalKey) {
       updates[edit.originalKey ?? ''] = undefined
     }
-    setTranslations(getActiveI18n(), updates, true)
+    setTranslations({
+      code: getActiveI18n(),
+      translations: updates,
+      replace: true,
+      forceSave: true,
```

Comment:
I added this extra parameter since I realized `replace` doesn't trigger a save. So at the end of editing a translation, it wouldn't be saved until the next command was issued.